flags.css

```diff
@@ -1,17 +1,17 @@
+.flagContainer {
+    height: 66vmin;
+    width: 100vmin;
```

Comment:
this is going to lead to some problems.  100% of the minimum viewport dimension+ 10% of the minimum viewport dimension (5% *2) is going to be 110% of the minimum viewport dimension.